frontend/src/pages/finalReflection.js

```diff
@@ -45,17 +45,17 @@ function FinalReflection({ pages, setPages, activePage, setActivePage }) {
     setActivePage((prevPage) => "feedback");
   }
 
-  function goToResponse() {
-    if (pages.response.completed) {
-      if (pages.response.completed) {
-        if (!pages.response.visited) {
+  function goToConclusion() {
+    if (pages.conclusion.completed) {
+      if (pages.conclusion.completed) {
```

Comment:
Double `if`'s?

Reply:
The double ifs were already there but you are right, I should have noticed this. Thanks!